src/Umbraco.Abstractions/Services/IImageSharpImageService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Umbraco.Core.Models;
+
+namespace Umbraco.Core.Services
+{
+    public interface IImageSharpImageService
```

Comment:
`IImageSharpService` Why would you create an interface specific to a specific library then abstract out the model? 
This makes no sense.